network-container/mongodb-atlas-networkcontainer.json

```diff
@@ -0,0 +1,94 @@
+{
+    "typeName": "MongoDB::Atlas::NetworkContainer",
+    "description": "This resource allows to create, read, update and delete a network container",
+    "sourceUrl": "https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/master/network-container",
+    "definitions": {
+        "apiKeyDefinition": {
+            "type": "object",
+            "properties": {
+                "PublicKey": {
+                    "type": "string"
+                },
+                "PrivateKey": {
+                    "type": "string"
+                }
+            },
+            "additionalProperties": false
+        }
+    },
+    "properties": {
+        "ProjectId": {
+            "description": "The unique identifier of the project.",
+            "type": "string"
+        },
+        "ContainerId": {
+            "description": "Unique identifier of the Atlas VPC container for the AWS region.",
+            "type": "string"
+        },
+        "RegionName": {
+            "description": "Name of region",
+            "type": "string"
+        },
+        "Provisioned": {
+            "description": "Flag that indicates if the project has clusters deployed in the Network Peering container.",
+            "type": "boolean"
+        },
+        "ProviderName": {
+            "description": "The name of the provider",
+            "type": "string"
+        },
+        "VpcId": {
+            "description": "Unique identifier of the peer VPC.",
+            "type": "string"
+        },
+        "AtlasCidrBlock": {
+            "description": "CIDR block that Atlas uses for your clusters.",
+            "type": "string"
+        },
+        "Id": {
+            "description": "The CIDR block that Atlas uses for the clusters in your project.",
```

Comment:
copy pasta error? I'm not actually sure what Id this would refer to so can't help with suggested text